cpp/src/parquet/properties.h

```diff
@@ -174,6 +175,16 @@ class PARQUET_EXPORT ColumnProperties {
 
   void set_compression_level(int compression_level) {
     compression_level_ = compression_level;
```

Comment:
Can we just remove `compression_level_`?

Reply:
+1, I think we can just keep the codec options.